shell/src/main/java/org/apache/accumulo/shell/commands/CreateNamespaceCommand.java

```diff
@@ -62,12 +63,11 @@ public int execute(final String fullCommand, final CommandLine cl, final Shell s
       }
     }
     if (configuration != null) {
-      for (Entry<String,String> entry : configuration.entrySet()) {
-        if (Property.isValidTablePropertyKey(entry.getKey())) {
-          shellState.getAccumuloClient().namespaceOperations().setProperty(namespace,
-              entry.getKey(), entry.getValue());
-        }
-      }
+      var propsToAdd = configuration.entrySet().stream()
+          .filter(entry -> Property.isValidTablePropertyKey(entry.getKey()))
+          .collect(Collectors.toMap(Entry::getKey, Entry::getValue));
+      shellState.getAccumuloClient().namespaceOperations().modifyProperties(namespace,
+          properties -> properties.putAll(propsToAdd));
```

Comment:
```suggestion
      final Map<String,String> config = configuration;
      shellState.getAccumuloClient().namespaceOperations().modifyProperties(namespace,
          properties -> config.entrySet().stream()
              .filter(entry -> Property.isValidTablePropertyKey(entry.getKey()))
              .forEach(entry -> properties.put(entry.getKey(), entry.getValue())));
```
Basically the same suggestion as in #2965